### PR TITLE
GATEWAY-3073

### PR DESCRIPTION
**Pull request by bhumphrey**
DR 2.0 and 3.0 inbound only

### CODE REVIEW

File: Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/orchestration/AbstractPassThroughOrchestratable.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        super(pt, at);
+    }
+
+    /**
+     * This is always pass through.
+     * @returns true
+     */
+    @Override
+    final public boolean isPassthru() {
+        return false;
```
Comment (akong) [inline Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/orchestration/AbstractPassThroughOrchestratable.java]: Shouldn't this be returning true?

Comment (bhumphrey) [inline Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/orchestration/AbstractPassThroughOrchestratable.java]: yes it should.

File: Product/Production/Common/Properties/Dev/DocumentRetrieveProxyConfig.xml [hunk 2]
```diff
@@ -91,24 +91,6 @@
...
-
-    <alias alias="nhindocretrieve" name="nhindocretrievewssecured" />
-
-    <!-- No-op Implementation -->
-	<bean lazy-init="true" class="gov.hhs.fha.nhinc.docretrieve.nhin.proxy.NhinDocRetrieveProxyNoOpImpl" id="nhindocretrievenoop" name="nhindocretrievenoop"> 
-		<meta key="impltype" value="noop"/>
-	</bean> 
-
-	<!-- Secured web service Implementation -->
-    <bean lazy-init="true" class="gov.hhs.fha.nhinc.docretrieve.nhin.proxy.NhinDocRetrieveProxyWebServiceSecuredImpl" id="nhindocretrievewssecured" name="nhindocretrievewssecured">
```
Comment (akong) [inline Product/Production/Common/Properties/Dev/DocumentRetrieveProxyConfig.xml:107]: Wait... why are we deleting the bean for the Nhin Proxies?  We should delete the passthrough ones.

File: Product/Production/Gateway/DocumentRetrieve_20/src/test/java/gov/hhs/fha/nhinc/docretrieve/inbound/DocRetrieveTest.java [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+import org.junit.Test;
+
+/**
+ * @author bhumphrey
+ * 
+ */
+public class DocRetrieveTest {
+
+    @Test
+    public void bazinga() {
```
Comment (akong) [inline Product/Production/Gateway/DocumentRetrieve_20/src/test/java/gov/hhs/fha/nhinc/docretrieve/inbound/DocRetrieveTest.java]: Sheldon?

File: Product/Production/Services/DocumentRetrieveCore/src/main/java/gov/hhs/fha/nhinc/docretrieve/nhin/InboundDocRetrievePolicyTransformer_g0.java [hunk 4]
```diff
@@ -18,24 +20,69 @@
...
-            PolicyEngineChecker policyChecker = new PolicyEngineChecker();
-            policyReq = policyChecker.checkPolicyDocRetrieve(policyCheckReq);
+        if (message instanceof InboundDocRetrieveOrchestratable) {
+            policyReq = transformInbound((InboundDocRetrieveOrchestratable) message);
         }
         return policyReq;
     }
 
+    /**
+     * To be removed!
```
Comment (akong) [inline Product/Production/Services/DocumentRetrieveCore/src/main/java/gov/hhs/fha/nhinc/docretrieve/nhin/InboundDocRetrievePolicyTransformer_g0.java:47]: Remove?

File: Product/Production/Common/Properties/Dev/DocumentRetrieveProxyConfig.xml (hunk 2 above)
Comment (bhumphrey) [inline Product/Production/Common/Properties/Dev/DocumentRetrieveProxyConfig.xml:107]: over excited deleting. will fix.

**Pull request merged**